python/setup.py

```diff
@@ -371,14 +381,33 @@ def _add_extra_compile_arg(self, flag, compile_args):
     def add_debug_or_release_flags(self):
         """Return compiler flags for Release and Debug build types."""
         if NGRAPH_PYTHON_DEBUG in ['TRUE', 'ON', True]:
-            return ['-O0', '-g']
+            if sys.platform == 'win32':
+                return ['/Od', '/Zi', '/RTC1']
+            else:
+                return ['-O0', '-g']
         else:
-            return ['-O2', '-D_FORTIFY_SOURCE=2']
+            if sys.platform == 'win32':
+                return ['/O2']
+            else:
+                return ['-O2', '-D_FORTIFY_SOURCE=2']
+
+    def _add_win_compiler_flags(self, ext):
+        self._add_extra_compile_arg('/GL', ext.extra_compile_args)  # Whole Program Optimization
+        self._add_extra_compile_arg('/analyze', ext.extra_compile_args)
+
+    def _add_unix_compiler_flags(self, ext):
+        if not self._add_extra_compile_arg('-fstack-protector-strong', ext.extra_compile_args):
+            self._add_extra_compile_arg('-fstack-protector', ext.extra_compile_args)
+
+        self._add_extra_compile_arg('-fvisibility=hidden', ext.extra_compile_args)
+        self._add_extra_compile_arg('-flto', ext.extra_compile_args)
```

Comment:
It would be great if you would find equivalent flag to `-flto` (aka _link time optimizations_) on MSVC. I remember that the overall whl size was reduced about 30-40% just by using this flag.

Reply:
I've found `/GL` which is whole program optimization. I don't know the difference in sizes of the package with or without this flag though.